docs/site/Discovering-models.md

```diff
@@ -0,0 +1,45 @@
+---
+lang: en
+title: 'Discovering models from relational databases'
+keywords: LoopBack 4.0, LoopBack-Next
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Discovering-models.html
+---
+
+## Synopsis
+
+LoopBack makes it simple to create models from an existing relational database.
+This process is called _discovery_ and is supported by the following connectors:
+
+- [Cassandra](Cassandra-connector.html)
+- [MySQL](MySQL-connector.html)
+- [Oracle](Oracle-connector.html)
+- [PostgreSQL](PostgreSQL-connector.html)
+- [SQL Server](SQL-Server-connector.html)
+- [IBM DB2](DB2-connector.html)
+- [IBM DashDB](DashDB.html)
+- [IBM DB2 for z/OS](DB2-for-z-OS.html)
+- [SAP HANA](https://www.npmjs.org/package/loopback-connector-saphana) - Not
+  officially supported; see [Community connectors](Community-connectors.html).
+
+## Overview
+
+Models can be discovered from a supported datasource by running the
+`lb4 discover` command.
+
+**The LoopBack project must be built and contain the built datasource files in
+`PROJECT_DIR/dist/src/datasources.js`**
```

Comment:
I don't think LB4 projects have a file called `dist/src/datasources.js`. Did you mean `dist/src/datasources/*.js` instead?

Also please note that we have recently changed the organization of project files in the `dist` folder, the compiler is configured to compile `src/*` to `dist/*`, i.e. `src/datasources/db.datasource.ts` is compiled to `dist/datasources/db.datasource.js` now.

Please update the README (and the implementation?) accordingly.

Reply:
Yes, that's what I meant.  Fixed